src/mixins/animation-frame.html

```diff
@@ -0,0 +1,64 @@
+<script>
+  window.requestAnimFrame = (function(){
+    return  window.requestAnimationFrame ||
+      window.webkitRequestAnimationFrame ||
+      window.mozRequestAnimationFrame    ||
+      function( callback ){
+        window.setTimeout(callback, 1000 / 60);
+      };
+  })();
+
+  const AnimationFrame = subclass => class extends subclass {
```

Comment:
Class name does not describe the content. Maybe would be better `ScrollFunctions` or something like that